onos-classic/values.yaml

```diff
@@ -28,6 +28,8 @@ resources:
     cpu: 0.5
     memory: 512Mi
 
+exposeNodePorts: false
```

Comment:
We are exposing NodePort in both cases. Is `ExposeIndividualOpenFlow` a better name for this?

Reply:
If deployed with this to false and nothing is set for `onosSshPort` or `onosApiPort` we expose no node ports. I'll change it to individualOpenFlowNodePorts